addons/website/views/website_templates.xml

```diff
@@ -247,6 +250,12 @@
 </template>
 
 <!-- Navbar options -->
+<template id="option_layout_hide_header" inherit_id="website.layout" active="False">
+    <xpath expr="//header" position="attributes">
+        <attribute name="t-attf-class" add="d-none" separator=" "/>
+    </xpath>
+</template>
```

Comment:
Since it is an option which requires a reload anyway, can't we just remove the header entirely instead of d-none for this specific option?

Same remark for every option like that. Also, instead of an option which hides the header, would this be possible to extract the header in a template to have a xml template which *adds* the header. That way, any xpath targeting the header should inherit from this template and it could be safely disabled.

Reply:
As discussed I will set the "no_header" to do it